src/shared/components/challenge-detail/Winners/index.jsx

```diff
@@ -78,7 +78,8 @@ export default function Winners(props) {
                   <a
                     styleName="download"
                     target="_blank"
-                    href={w.submissionDownloadLink}
+                    href={`${config.URL.STUDIO}/` +
+                          `?module=DownloadSubmission&sbmid=${submissionId}`}
```

Comment:
@shubhendusaurabh The new links seems to work fine for design challenges, but does not work for development ones. I guess, you should update it to use original link for dev challenges, and the last one for design. Also, you should not use `+` to concatentate strings (it actually should cause a ESLint error).